ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml

```diff
@@ -55,20 +55,13 @@ StackLayout {
                 StatusSmartIdenticon {
                     objectName: "communityOverviewSettingsPanelIdenticon"
                     name: root.name
-
-                    icon {
-                        width: 80
-                        height: 80
-                        isLetterIdenticon: !root.logoImageData
-                        color: root.color
-                        letterSize: width / 2.4
-                    }
-
-                    image {
-                        width: 80
-                        height: 80
-                        source: root.logoImageData
-                    }
+                    asset.width: 80
+                    asset.height: 80
+                    asset.isLetterIdenticon: !root.logoImageData
+                    asset.color: root.color
+                    asset.letterSize: width / 2.4
+                    asset.source: root.logoImageData
+                    asset.isImage: !!root.logoImageData
```

Comment:
remove !!? not very efficient?
root.logoImageData  !== ""  is better?

Reply:
Removed, however I think !! works in both undefined and empty cases, isn't it?

Reply:
I am not sure :?

Reply:
What @alexandraB99 suggested is correct. While `!== ""` checks for non empty string `!!expr` checks for truthyness
A non-empty string is falsy in JS. So is `undefined` and `null`